pkg/config/config.go

```diff
@@ -150,3 +156,27 @@ func Finalize(pArgs *ProgArgs) error {
 	}
 	return err
 }
+
+func ValidateConfigRootPath(configRoot string) (string, error) {
+	allowedPrefixes := []string{
+		"/etc/rte",
+		"/run/rte",
+		"/var/rte",
+		"/usr/local/etc/rte",
+		"/home", // mostly for tests
```

Comment:
ok, but then should be something like `/home/*/.rte`

Reply:
This is going to break the existing unit tests